app/main.py

```diff
@@ -1,5 +1,81 @@
 class Cargo:
-    def __init__(self, weight: int) -> None:
+    def __init__(
+            self,
+            weight: int
+    ) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(
+            self,
+            name: str,
+            weight: int,
+            coords: None | list = None
+    ) -> None:
+
+        if coords is None:
+            coords = [0, 0]
+
+        self.name = name
+        self.weight = weight
+        self.coords = coords
```

Comment:
Avoid using if condition, perform assignment in one line using either if condition 
```suggestion
        self.name = name
        self.weight = weight
        self.coords = coords if coords else [0, 0]
```
or `or` expression
```suggestion
        self.name = name
        self.weight = weight
        self.coords = coords or [0, 0]
```